qa/workunits/ceph-helpers.sh

```diff
@@ -1297,6 +1297,48 @@ function test_wait_for_clean() {
 #######################################################################
 
 ##
+# Wait until the cluster becomes HEALTH_OK again or if it does not make progress
+# for $TIMEOUT seconds.
+#
+# @return 0 if the cluster is HEALTHY, 1 otherwise
+#
+function wait_for_health() {
+    local grepstr=$1
+    local -a delays=($(get_timeout_delays $TIMEOUT .1))
+    local -i loop=0
+
+    while ! ceph health detail | grep "$grepstr" ; do
+        ceph health detail
+	if (( $loop >= ${#delays[*]} )) ; then
+            return 1
+        fi
+        sleep ${delays[$loop]}
+        loop+=1
+    done    
+    return 0
```

Comment:
if `ceph health detail | grep "$grepstr"` returns 0, we don't need to `return 0` explicitly.